pkg/types/validation/installconfig.go

```diff
@@ -135,8 +135,7 @@ func ValidateInstallConfig(c *types.InstallConfig, usingAgentMethod bool) field.
 			allErrs = append(allErrs, field.Forbidden(field.NewPath("arbiter"), fmt.Sprintf("%s feature must be enabled in order to use arbiter cluster deployment", features.FeatureGateHighlyAvailableArbiter)))
 		}
 	}
-	multiArchEnabled := types.MultiArchFeatureGateEnabled(c.Platform.Name(), c.EnabledFeatureGates())
```

Comment:
Also let's remove this function (as this is the only use):

https://github.com/openshift/installer/blob/25b5d2154bf53fac3d8e81f6f08e6630ae9d198b/pkg/types/installconfig.go#L622-L633